### ISSUE TITLE
Feature Request: Move game ID to the end of installation directory names

### ISSUE THREAD

**Issue by kjack9** [Phalcode/gamevault-app#448]
**Is your feature request related to a problem? Please describe.**
When I have many games installed, it is often difficult to find the exact folder a game is sorted in because they are prepended with the game's GameVault ID.

**Describe the solution you'd like**
Move the Game ID to the end of the game's install directory name.

**Describe alternatives you've considered**
I'm less picky on the exact formatting, mostly interested in the game name being the sorting factor in a file browser rather than anything else.

**Additional context**
`E:\Games\GameVault\Installations\(11)Astroneer\` becomes `E:\Games\GameVault\Installations\Astroneer (11)\`

**Comment by Alfagun74** [comment]
I think we will remove these IDs in folder-names entirely once we revamp the install locations via ticket https://github.com/Phalcode/gamevault-app/issues/229, but we won't implement this on its own right now because it would interfere with the game naming scheme (year at the end of the filename).

**Comment by GlassedSilver** [comment]
Wait, that means if users have multiple games with the same name the only way to deal with naming conflicts would be to create n library roots?

That sounds like a pretty poor UX, but maybe I'm not getting this right.

**Comment by Alfagun74** [comment]
@GlassedSilver No, I meant that the folder name should be totally irrelevant by then, there should be multiple install locations you can install to and scan from, and the true discriminator of games will be a YAML/metadata file inside the install location that GameVault will look for which will contain the ID.

**Comment by GlassedSilver** [comment]
Fair point and I got that, but the issue remains that to install two same-named games you would be required to either have to library roots or define an alt-name for the folder manually I guess.

If there could be an option to define the naming scheme for the install target folder that'd be amazing, MAYBE even allow a steam app id as variable.

**Comment by Alfagun74** [comment]
> ... the issue remains that to install two same-named games you would be required to either have to library roots or define an alt-name for the folder manually I guess.

Well surprise, surprise, What a groundbreaking discovery. 

We'll just autogenerate:   

/Game/  
/Game (1)/  
   
Then I guess if there are conflicts.

**Comment by GlassedSilver** [comment]
But then the path won't be predictable, for example if you want to run scripts on your library across reinstallazions or devices.
